nodeup/pkg/model/containerd.go

```diff
@@ -51,103 +51,15 @@ var containerdVersions = []packageVersion{
-		Source:         "https://download.docker.com/linux/debian/dists/buster/pool/stable/amd64/containerd.io_1.2.10-3_amd64.deb",
-		Hash:           "355973b23d3d172b5cfb05bc605f2b0cd7145f2fcc572264225d8910701c650d",
-	},
-
-	// 1.2.10 - Ubuntu Xenial
-	{
-		PackageVersion: "1.2.10",
-		Name:           "containerd.io",
-		Distros:        []distros.Distribution{distros.DistributionXenial},
-		Architectures:  []architectures.Architecture{architectures.ArchitectureAmd64},
-		Version:        "1.2.10-3",
-		Source:         "https://download.docker.com/linux/ubuntu/dists/xenial/pool/stable/amd64/containerd.io_1.2.10-3_amd64.deb",
-		Hash:           "ec7f4698df82c4dc683a8835ef5a3ecce4df1e41c5d65b4e17558cdccf44952b",
-	},
-
-	// 1.2.10 - Ubuntu Bionic
-	{
-		PackageVersion: "1.2.10",
-		Name:           "containerd.io",
-		Distros:        []distros.Distribution{distros.DistributionBionic},
-		Architectures:  []architectures.Architecture{architectures.ArchitectureAmd64},
-		Version:        "1.2.10-3",
-		Source:         "https://download.docker.com/linux/ubuntu/dists/bionic/pool/stable/amd64/containerd.io_1.2.10-3_amd64.deb",
-		Hash:           "ffa99f3f4b3a76c66f04ad327b8eb6b437e311887e3e0330447fbf5ea2ddd827",
-	},
-
-	// 1.2.10 - CentOS7 / Rhel 7
-	{
-		PackageVersion: "1.2.10",
-		Name:           "containerd.io",
-		Distros:        []distros.Distribution{distros.DistributionRhel7, distros.DistributionCentos7},
-		Architectures:  []architectures.Architecture{architectures.ArchitectureAmd64},
-		Version:        "1.2.10",
-		Source:         "https://download.docker.com/linux/centos/7/x86_64/stable/Packages/containerd.io-1.2.10-3.2.el7.x86_64.rpm",
-		Hash:           "8f29304480a197343fb6fb0fd28e8753aed4e177545568cb4ea3b0f0c02ebf82",
-	},
-
-	// 1.2.10 - CentOS8 / Rhel 8
-	{
-		PackageVersion: "1.2.10",
-		Name:           "containerd.io",
-		Distros:        []distros.Distribution{distros.DistributionRhel8, distros.DistributionCentos8},
-		Architectures:  []architectures.Architecture{architectures.ArchitectureAmd64},
-		Version:        "1.2.10",
-		Source:         "https://download.docker.com/linux/centos/7/x86_64/stable/Packages/containerd.io-1.2.10-3.2.el7.x86_64.rpm",
-		Hash:           "8f29304480a197343fb6fb0fd28e8753aed4e177545568cb4ea3b0f0c02ebf82",
-	},
-
 	// 1.2.10 - Linux Generic
-	//
-	// * AmazonLinux2: the Centos7 package depends on container-selinux, but selinux isn't used on amazonlinux2
-	// * UbuntuFocal: no focal version available at download.docker.com
 	{
 		PackageVersion: "1.2.10",
 		PlainBinary:    true,
-		Distros:        []distros.Distribution{distros.DistributionAmazonLinux2, distros.DistributionFocal},
 		Architectures:  []architectures.Architecture{architectures.ArchitectureAmd64},
 		Source:         "https://storage.googleapis.com/cri-containerd-release/cri-containerd-1.2.10.linux-amd64.tar.gz",
 		Hash:           "9125a6ae5a89dfe9403fea7d03a8d8ba9fa97b6863ee8698c4e6c258fb14f1fd",
 	},
 
-	// 1.2.11 - Linux Generic
```

Comment:
I presume you're planning on removing support for these four versions from 1.18 as well.

I notice we don't have API validation for the containerd version.

Reply:
Yes, will cherry-pick to 1.18 also. These were only used during development and current versions will be the final ones.

Reply:
The version validation for Docker was a bit rushed up when 1.x support was removed.
Thanks for the reminder to add validation for containerd also.